.github/workflows/package.yml

```diff
@@ -0,0 +1,49 @@
+name: package
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+  workflow_dispatch:
+
+env:
+  CI: false
```

Comment:
we aren't running the build outside the docker image, so this flag isn't required here